tailwind.config.js

```diff
@@ -11,4 +11,10 @@ module.exports = {
     require('@tailwindcss/forms'),
     require('@tailwindcss/typography'),
   ],
+  safelist: [
+    ...[...Array(101).keys()].flatMap(i => `w-[${i}%]`),
```

Comment:
Oh man, is that how it turns out in JavaScript? Well, could be worse I guess.

Reply:
yeah :frowning_face: